src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/HttpClientTransport.cs

```diff
@@ -72,10 +79,10 @@ public override void AddHeader(HttpHeader header)
             public override void AddHeader(string name, string value)
             {
                 // TODO (pri 1): any other headers must be added to content?
-                if (name.Equals("Content-Type", StringComparison.InvariantCulture)) {
+                if (name.Equals("Content-Type", StringComparison.InvariantCultureIgnoreCase)) {
```

Comment:
I filed https://github.com/Azure/azure-sdk-for-net/issues/5578 to deal with content headers.

Reply:
And I fixed it here, wasn't too hard.